### PR TITLE
Add active storage

### PR DESCRIPTION
**Pull request by Elhambasir**
## What is done in this PR?
In this PR, I installed active storage to handle image uploading. In order to make the project work with active storage I changed something in the doctor model. For example, I removed the photo column because we have it in the active storage table and no need to have it in the doctor table.

### CODE REVIEW

File: app/controllers/api/v1/doctors_controller.rb [hunk 1]
```diff
@@ -3,9 +3,9 @@ class Api::V1::DoctorsController < ApplicationController
 
   # GET /doctors
   def index
-    @doctors = Doctor.all
+    @doctors = Doctor.all.joins(:photo_attachment)
 
-    render json: @doctors
+    render json: @doctors.map { |doctor| doctor.as_json.merge({ photo_url: url_for(doctor.photo) }) }
```
Comment (Lul-Abdifan) [inline app/controllers/api/v1/doctors_controller.rb:8]: - Could you please look at the screenshot attached below .The form data is not being added to the database.

Screenshot from 2023-08-29 14-36-49

File: app/models/doctor.rb [hunk 2]
```diff
@@ -3,5 +3,18 @@ class Doctor < ApplicationRecord
   # Validations will be added here
   validates :name, presence: true
```
Comment (Lul-Abdifan) [inline app/models/doctor.rb:4]: - Please kindly make this code on one line

**Review by Lul-Abdifan — CHANGES_REQUESTED** [review]
# Required Changes :recycle:
Ukhv

## Hey @Elhambasir  :wave:

## Highlights :1st_place_medal:

✔️ Descriptive PR Title
✔️ Descriptive PR Summary
✔️ Following Gitflow
✔️ Following some requirements
✔️ Great work on adding the Add Doctor functionality 👍


#### Changes Required
✔️ Linters are not passing
✔️ See below for other changes

Good job on implementing some of the requirements, there are some issues that you still need to work on to go to the next project but you are almost there! :blush:

# Optional

_All the changes with optional tags are not crucial enough to prevent you from getting the approval but we highly recommend to students that they should take these changes into consideration._

Happy coding! :clap: :clap: :clap:

Feel free to leave any questions or comments in the PR thread if something is not 100% clear. Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

---------------------------------------

_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by sumon766 — APPROVED** [review]
💐 Hi @Elhambasir, 💐

You have done a great job implementing the requirements for the project 🎉. Your project is complete, there is nothing else to say, other than it's time to merge it. Congratulations...🥳👏

giphy

**Pull request merged**